src/main/scala/kanaloa/reactive/dispatcher/metrics/StatsDMetricsCollectorSettings.scala

```diff
@@ -0,0 +1,11 @@
+package kanaloa.reactive.dispatcher.metrics
+
+sealed trait MetricsCollectorSettings
+
+case class StatsDMetricsCollectorSettings(
+  host:             String,
+  namespace:        String = "ReactiveDispatchers",
```

Comment:
minor thing -- default should probably be `reactiveDispatchers` (lowercase r) to be consistent with StatsD naming conventions